lib/core/card-view/validators/card-view-item-int.validator.ts

```diff
@@ -22,6 +22,6 @@ export class CardViewItemIntValidator implements CardViewItemValidator {
     message = 'CORE.CARDVIEW.VALIDATORS.INT_VALIDATION_ERROR';
 
     isValid(value: any): boolean {
-        return !isNaN(value) && (function(x) { return (x | 0) === x; })(parseFloat(value));
+        return !value || !isNaN(value) && (function(x) { return (x | 0) === x; })(parseFloat(value));
```

Comment:
can we avoid creating a new function each time `isValid` is requested? looks like a useless thing here to be honest.